volume17.xml

```diff
@@ -0,0 +1,309 @@
+            </div>
+            <div xml:id="EBA-1911-03-07" type="entry">
+                <dateline>
+                    <date when="1911-03-07">Tuesday. Mar. 7th.</date>
+                </dateline>
+                <p>Started at 6 A.M. and went on until 6 P.M. - 5 miles from <placeName ref="#Assiut">Assiut</placeName> &#8211; making 87 miles.</p>
+            </div>
+            <div xml:id="EBA-1911-03-08" type="entry">
+                <dateline>
+                    <date when="1911-03-08">Wednesday. Mar. 8th.</date>
+                </dateline>
+                <p>52 miles. Arrived at <placeName ref="#Assiut">Assiut</placeName> early, and waited for various things &#8211; 4 hours &#8211; then on to 7 miles of <placeName ref="#Hadj_Kandul">Hadj Kandul</placeName>.</p>
+            </div>
+            <div xml:id="EBA-1911-03-09" type="entry">
+                <dateline>
+                    <date when="1911-03-09">Thursday. March 9th.</date>
+                </dateline>
+                <p>Started at 7 o'clock &#8211; stopped at <placeName ref="#Hadj_Kandul">Hadj Kandul</placeName> for an hour <pb n="82"/>so that <persName ref="#Rathbone_Elena">El&#232;na</persName> should see the Khuenaton pavement, then on, until <placeName ref="#Beni_Hassan">Beni Hassan</placeName> was reached, which <persName ref="#Rathbone_Elena">El&#232;na</persName> and <persName ref="#Burton_Harry">Harry</persName> visited. The day hot &#8211; wind and clouds of dust. 43 miles.</p>
```

Comment:
"Khuenaton pavement" could be an interesting popup—should we consider a tag for art objects?

I assume she's referring to something like this: https://www.metmuseum.org/art/collection/search/548432

Reply:
This is a good idea. The link is for flooring from Amenhotep III's palace; he was the father of Akhenaten who is referenced here by Emma as 'Khuenaton'.  We could use the tag <name type="artwork" ref="Akhenaten pavement">. I would like to distinguish between the Italian art and Egyptian - maybe 'ancient artwork' and 'Renaissance artwork' or something like that?